kuttl-test.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
This file shouldn't be committed.  In fact, it should probably be added to .gitignore